tests/FileUpload/FileNameGenerator/MD5Test.php

```diff
@@ -11,16 +11,16 @@ class MD5Test extends TestCase
 {
     protected $filesystem;
 
-    public function setUp()
+    protected function setUp()
```

Comment:
Can you revert this back to making use of `public` access? It's the reason why the tests are failing? 


Applies to others

Reply:
Fixed.

I've changed protected to public.
But there was a different problem:

I've changed `script: phpunit` to  `script: vendor/bin/phpunit`

The problem was that the wrong version of phpunit was used.

thank you